core/src/test/java/org/testcontainers/containers/VncRecordingContainerTest.java

```diff
@@ -0,0 +1,33 @@
+package org.testcontainers.containers;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class VncRecordingContainerTest {
+
+    private GenericContainer target;
+
+    @Before
+    public void initialize() {
+        target = new GenericContainer("selenium/standalone-chrome-debug").
+            withNetwork(Network.SHARED).
+            withExposedPorts(5900);
+        target.start();
+    }
+
+    @Test
+    public void verifyStartup() {
+        try (VncRecordingContainer container = new VncRecordingContainer(target)) {
+            container.start();
+            assertThat(container.isRunning()).isTrue();
```

Comment:
I wonder if this test is needed, since we have other tests that assert the recording